packages/amplify-cli/src/__tests__/extensions/amplify-helpers/git-manager.test.ts

```diff
@@ -0,0 +1,22 @@
+import * as fs from 'fs-extra';
+import { insertAmplifyIgnore } from '../../../extensions/amplify-helpers/git-manager';
+jest.mock('fs-extra');
+
+const fsMock = fs as jest.Mocked<typeof fs>;
+
+fsMock.readFileSync.mockImplementation(() => 'test');
+fsMock.writeFileSync.mockImplementation();
+
+describe('git-manager', () => {
+  const gitIgnoreFilePath = '';
+  test('append files should be excluded to .gitignore', () => {
+    fsMock.existsSync.mockImplementation(() => true);
+    insertAmplifyIgnore(gitIgnoreFilePath);
+    expect(fsMock.appendFileSync).toHaveBeenCalled();
```

Comment:
It might be worth validating the arguments passed to `appendFileSync()` and `writeFileSync()`.